oracle/config/app.go

```diff
@@ -175,15 +191,27 @@ func ReadConfigFromAppOpts(opts servertypes.AppOptions) (AppConfig, error) {
 
 	// get the price ttl
 	if v := opts.Get(flagPriceTTL); v != nil {
-		if cfg.PriceTTL, err = cast.ToDurationE(v); err != nil {
-			return cfg, err
+		priceTTL, err := cast.ToDurationE(v)
+		if err != nil {
+			return cfg, fmt.Errorf("price ttl must be a positive duration")
+		}
```

Comment:
this won't return an error in negative durations, i.e. `cast.ToDuration("-5s")`

Reply:
otherwise invalid so imo its fine if we replace. its a bit of a footgun but i cant imagine why someone would do that (code would prevent this).